### ISSUE TITLE
Crash 

### ISSUE THREAD

**Issue by wiqg** [MrCrayfish/MrCrayfishVehicleMod#47]
https://pastebin.com/GudnimQB
I think its conflict with BlankPlanet because everything was OK until i installed it.

**Comment by JanHolger** [comment]
This looks like an incompatibility issue. Have you tried updating Obfuscate? From the crash log i can see that you are using 0.2.5 but the current available version is 0.2.6 (released on 7. of october).

**Comment by wiqg** [comment]
Yeah , now starts the game and right when i join my world it instantly crashes...
